torch/nn/functional.py

```diff
@@ -3016,6 +3020,8 @@ def cross_entropy(
         >>> loss = F.cross_entropy(input, target)
         >>> loss.backward()
     """
+    if dim is not None:
+        input = input.swapaxes(dim, 1)
```

Comment:
The contract we provide for `__torch_function__` support is that the full Python API should be overrideable.

Given this, the `handle_torch_function()` logic should be first (before the lines added here), and `dim=dim` should be passed to the `handle_torch_function()` call.

Reply:
@jbschlosser, Alright, I'll modify this.